notebooks/214-grammar-correction/214-grammar-correction.ipynb

```diff
@@ -27,23 +27,29 @@
     "- Install prerequisites\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    to_quantize = widgets.Dropdown(
</pre><p>maybe checkbox will be better here then dropdown as you need to choose only between 2 options. and default value is better to set to enable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Done